src/peft/utils/constants.py

```diff
@@ -120,6 +120,42 @@ def starcoder_model_postprocess_past_key_value(past_key_values):
     "qwen2": ["q_proj", "v_proj"],
 }
 
+TRANSFORMERS_MODELS_TO_MOSLORA_TARGET_MODULES_MAPPING = {
```

Comment:
Can we just make this a copy of `TRANSFORMERS_MODELS_TO_LORA_TARGET_MODULES_MAPPING`? I think each time that LoRA works, MosLoRA should also work.